### ISSUE TITLE
Idea for Wenyan Markdown

### ISSUE THREAD

**Issue by Meowcolm024** [wenyan-lang/wenyan#572]
As we have a Wenyan json, maybe we could have a Wenyan markdown OwO. 
Even though Markdown's syntax is simple enough, it's still interesting to create a Wenyan version.
I've made a prototype: [箋Jian](https://github.com/Meowcolm024/jian)

**Comment by LingDong-** [comment]
I like the idea! Thanks for the proposal. We can even use it for the wenyan handbook project I'm working on.

Also, we can use 〇 〇 〇 or 一 一 一 for unordered lists, seems to be common in ancient prints.

We might also indicate title structure with indentation which is also common, but might lead to readability problems. Let me think about this.

![Snip20200202_7](https://user-images.githubusercontent.com/7929704/73613436-b53f8680-45c3-11ea-8b89-acf58e4f45a1.png)

**Comment by Meowcolm024** [comment]
I'm wondering how to represent **Bold** and *Italic* in it (and there is no such thing as *italic* in Chinese, maybe we can just ignore that).
Terminologies for tables and code blocks are also not decided... 🙃

**Comment by Meowcolm024** [comment]
@LingDong- 
I've tried to use indentations for title structure. It's not that confusing when reading, but it might confuse the parser (with the body paragraphs). So I have to use punctuations to distinguish them.

**Comment by iensu** [comment]
For numbered lists shouldn't you be using 天干地支? Like 
甲、文言也
乙、Haskell也
丙、表之實例也

Or was this only used for dates? I'm not as well versed in acient Chinese as I'd like to be :D

**Comment by Meowcolm024** [comment]
@iensu 
If we use `天干地支` we can only reach as much as `60`(probably enough for ordered list). 
And if we only use `天干` only, it would be `甲乙丙丁戊己庚辛壬癸`(which is 10...). 
And it's good idea by the way.
Besides, I think that `天干地支` is mostly used in dates, ranking and organic chemistry...

**Comment by XingZiLong** [comment]
Generally 天干 can be  used (and usually be used) as a sort of order but 天干地支 can’t — we don’t use it unless to describe time yearly. 地支 is often used to describe time hourly.

**Comment by Meowcolm024** [comment]
That's correct. In addition, 天干地支 is also used to describe time monthly and daily.
But 天干 is mostly used in order and organic chemistry nowadays.

**Comment by antfu** [comment]
I have added a link to the project in our [wiki page](https://github.com/wenyan-lang/wenyan/wiki/Related-Tools). Thanks for the awesome work and looking forward to it :)
